pydrawise/client.py

```diff
@@ -266,3 +268,56 @@ async def delete_zone_suspension(self, suspension: ZoneSuspension) -> None:
             id=suspension.id
         ).select()
         await self._mutation(selector)
+
+    async def get_sensors(self, controller: Controller) -> list[Sensor]:
+        """Retrieves sensors associated with the given controller.
+
+        :param controller: Controller whose sensors to fetch.
+        :rtype: list[Sensor]
+        """
+        selector = self._schema.Query.controller(controllerId=controller.id).select(
+            self._schema.Controller.sensors.select(
+                *get_selectors(self._schema, Sensor)
+            ),
+        )
+
+        result = await self._query(selector)
+        return deserialize(list[Sensor], result["controller"]["sensors"])
+
+    async def get_water_flow_summary(
+        self, controller: Controller, sensor: Sensor, start: DateTime, end: DateTime
```

Comment:
`start` and `end` should be type `datetime` not `DateTime`.

Reply:
Fixed.